Sources/active-win/main.swift

```diff
@@ -1,6 +1,33 @@
 import AppKit
 
-// This shows the system prompt if there's no permission.
+// Uses AppleScript to get the browser URL from Chrome or Safari
+func getBrowserURL(_ appName: String) -> String? {
+	// Gets the appropriate AppleScript for each browser type
+	guard let scriptText = getScriptText(appName) else { return nil }
+	
+	// Prepare and execute AppleScript script to query URL
+	var error: NSDictionary?
+	guard let script = NSAppleScript(source: scriptText) else { return nil }
+	guard let outputString = script.executeAndReturnError(&error).stringValue else { return nil }
+	
+	return outputString
+}
+
+// Formats the AppleScript text for Chrome and Safari
+func getScriptText(_ appName: String) -> String? {
+	switch appName {
+	case "Google Chrome":
+		return "tell app \"Google Chrome\" to get the url of the active tab of window 1"
```

Comment:
Can you handle the new Chromium-based Edge browser? It uses Chromium, so should be exactly the same except for the name.

Reply:
```swift
case "Google Chrome", "Brave":
	return "tell app \"\(appName)\" to get the url of the active tab of window 1"
```

Reply:
Actually the apple script can be simplified to:

```
tell app "Google Chrome" to get URL of active tab of front window
```

Reply:
Updated all of the above with the most recent commit.